src/main/java/com/epam/controller/ReservationController.java

```diff
@@ -19,21 +19,25 @@ public ReservationController(ReservationService reservationService) {
         this.reservationService = reservationService;
     }
 
-    @GetMapping("/reservation")
-    public String listReservations(ModelMap modelMap) {
-        List<Reservation> list = reservationService.listReservations();
-        modelMap.addAttribute("listReservation", list);
+    @GetMapping
+    public String testadmin() {
+        return "redirect:/reservation/1";
+    }
+
+    @GetMapping("/{id}")
+    public String listReservations(@PathVariable Integer id, ModelMap modelMap) {
```

Comment:
Is it about properties? Please, remove this from the pull request and create another with this changes